WordPress/gutenberg
# Fix: Term Description block should only be available in the site editor

## What?

The term description block displays the description of taxonomies in archives, and does not output anything on the front 
if it is used in any other context. 

Closes https://github.com/WordPress/gutenberg/issues/38552

## Why?

It can be confusing if you add a block to the post content, that is visible in the editor but not the front.

## How?

Adds `__experimental": "fse"`, to block.json

## Testing Instructions
Create a new post using the block editor.
Try to add a term description block. Confirm that the block is not available.

Go to the Site Editor and edit any template. 
Try to add a term description block. Confirm that the block is available.
(if you view the front, it still wont show unless you are on an archive page for a taxonomy that has a description to show)
